rpm/SPECS/docker-ce-cli.spec

```diff
@@ -62,33 +62,33 @@ popd
 
 %install
 # install binary
-install -d $RPM_BUILD_ROOT/%{_bindir}
```

Comment:
@thaJeztah isn't `%{_binddir}` already pointing to the right directory and self-sufficient?

Reply:
Oups 😅